src/core/reference/src/op/interpolate.cpp

```diff
@@ -9,6 +9,67 @@
 namespace ov {
 namespace reference {
 
+std::function<int64_t(float, bool)> get_func(Nearest_mode mode) {
+    switch (mode) {
+    case Nearest_mode::ROUND_PREFER_CEIL:
+        return [](float x_original, bool) {
+            return static_cast<int64_t>(std::round(x_original));
+        };
+    case Nearest_mode::FLOOR:
+        return [](float x_original, bool) {
+            return static_cast<int64_t>(std::floor(x_original));
+        };
+    case Nearest_mode::CEIL:
+        return [](float x_original, bool) {
+            return static_cast<int64_t>(std::ceil(x_original));
+        };
+    case Nearest_mode::SIMPLE:
+        return [](float x_original, bool is_downsample) {
+            if (is_downsample) {
+                return static_cast<int64_t>(std::ceil(x_original));
+            } else {
+                return static_cast<int64_t>(x_original);
+            }
+        };
+    default:;
```

Comment:
```suggestion
    default:
        break;
```

Reply:
Resolved by moving return of default function here.